README.md

```diff
@@ -331,13 +331,13 @@ You can also check your proposal's validity using the [`validate-cbor`](#validat
 
 See the [Byron specification](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec) for more details on update proposals.
 
-You can submit your proposal using the `submit-byron-update-proposal` command.
+You can submit your proposal using the `byron submit-update-proposal` command.
 
 Example:
 ```
-cardano-cli submit-byron-update-proposal --config configuration/configuration-mainnet.yaml
-                                         --filepath my-update-proposal
-                                         --socket-path socket/0
+cardano-cli -- byron submit-update-proposal --config configuration/configuration-mainnet.yaml
```

Comment:
Path changed and now this is: `configuration/defaults/mainnet/configuration.yaml`